releases/release-1.27/release-notes/maps/pr-116119-map.yaml

```diff
@@ -0,0 +1,8 @@
+pr: 116119
+releasenote:
+  text: |-
+    ```
+    #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
+
+     <!--
+     This section can be blank if this pull request does not require a release note.
```

Comment:
Could be replaced with `None`. Same applied for all other files in this PR with the same text